script/OracleUpdater.s.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import {BaseScript, ChainId} from "utils/BaseScript.sol";
+import {ICauldronV2} from "interfaces/ICauldronV2.sol";
+import {MasterContract, OracleUpdater} from "periphery/OracleUpdater.sol";
+
+contract OracleUpdaterScript is BaseScript {
+    function deploy() public returns (OracleUpdater oracleUpdater) {
+        require(block.chainid == ChainId.Mainnet, "Wrong chain");
+
+        ICauldronV2[] memory cauldrons = new ICauldronV2[](26);
```

Comment:
what is the limit for inclusion here in terms of TVL?

Reply:
Just redeployed with a dynamic cauldron registry. So we can easily update it. Currently it should be V1 and V2 cauldrons with debt > 1000 MIM.